modules/apps/frontend-theme/frontend-theme-dialect/src/WEB-INF/frontend-token-definition.json

```diff
@@ -4906,6 +4906,2099 @@
+							"label": "tab-tertiary-vertical-item-border-bottom-width",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "tab-tertiary-vertical-item-border-bottom-width"
+								}
+							],
+							"name": "tabTertiaryVerticalItemBorderBottomWidth",
+							"type": "String"							
+						},
+						{
+							"defaultValue": "0",
+							"label": "tab-tertiary-vertical-item-border-left-width",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "tab-tertiary-vertical-item-border-left-width"
+								}
+							],
+							"name": "tabTertiaryVerticalItemBorderLeftWidth",
+							"type": "String"							
+						},
+						{
+							"defaultValue": "0",
+							"label": "tab-tertiary-vertical-item-border-right-width",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "tab-tertiary-vertical-item-border-right-width"
+								}
+							],
+							"name": "tabTertiaryVerticalItemBorderRightWidth",
+							"type": "String"							
+						},
+						{
+							"defaultValue": "0",
+							"label": "tab-tertiary-vertical-item-border-top-width",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "tab-tertiary-vertical-item-border-top-width"
+								}
+							],
+							"name": "tabTertiaryVerticalItemBorderTopWidth",
+							"type": "String"							
+						},
+						{
+							"defaultValue": "var(--spacer-3)",
+							"label": "tab-tertiary-vertical-item-padding-x",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "tab-tertiary-vertical-item-padding-x"
+								}
+							],
+							"name": "tabTertiaryVerticalItemPaddingX",
+							"type": "String"							
+						},
+						{
+							"defaultValue": "var(--spacer-3)",
+							"label": "tab-tertiary-vertical-item-padding-y",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "tab-tertiary-vertical-item-padding-y"
+								}
+							],
+							"name": "tabTertiaryVerticalItemPaddingY",
+							"type": "String"							
+						}
+					],
+					"label": "tertiary-vertical",
+					"name": "tertiary-vertical"
+				},
+				{
+					"frontendTokens": [
+						{
+							"defaultValue": "var(--color-neutral-1)",
+							"editorType": "ColorPicker",
+							"label": "tab-segment-background-color",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "tab-segment-background-color"
+								}
+							],
+							"name": "tabSegmentBackgroundColor",
+							"type": "String"							
+						},
+						{
+							"defaultValue": "var(--color-neutral-3)",
+							"editorType": "ColorPicker",
+							"label": "tab-segment-border-color",
+							"mappings": [
+								{
+									"type": "cssVariable",
+									"value": "tab-segment-border-color"
+								}
+							],
+							"name": "tabSegmentBorderColor",
+							"type": "String"							
+						},
+						{
+							"defaultValue": "2rem",
```

Comment:
The border-radius-pill variable is 50rem, should I use this one? there is another variable that is 1.5rem.

Reply:
Yep, that's the one. I think we should use that border-radius because our goal here is that we always have a pill shape for the segment, right? https://css-tricks.com/what-happens-when-border-radii-overlap/

Reply:
fixed